api.yaml

```diff
@@ -755,6 +755,15 @@ paths:
           $ref: "#/paths/~1room~1%7BroomToken%7D~1file~1%7BfileId%7D/get/responses/403"
         404:
           $ref: "#/paths/~1room~1%7BroomToken%7D~1file~1%7BfileId%7D/get/responses/404"
+  /user/inbox:
+    get:
+      tags: [Users]
+      summary: get all of the current user's inbox messages
+  /user/{sessionId}/inbox:
+    post:
+      tags: [Users]
+      summary: Submit an inbox message to another user
```

Comment:
Can use some more docs on what will be in the response.  It's definitely a bit tedious and massively over-indented but copy-and-pasting from one of the other ones works okay.

Reply:
yes, coming soon ™